src/main/java/org/reactome/addlinks/dataretrieval/ensembl/EnsemblBiomartRetriever.java

```diff
@@ -13,78 +12,57 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.nio.file.StandardOpenOption;
+import java.time.Duration;
 import java.util.*;
 
-public class EnsemblBiomartRetriever extends FileRetriever {
+public class EnsemblBioMartRetriever extends FileRetriever {
 
-    public EnsemblBiomartRetriever() { }
+    private Properties properties;
 
-    public EnsemblBiomartRetriever(String retrieverName)
+    public EnsemblBioMartRetriever() { }
+
+    public EnsemblBioMartRetriever(String retrieverName)
     {
         super(retrieverName);
     }
 
-    private static final int UNIPROT_QUERY_TOTAL = 2;
-    private static int successCount;
     private static final String  microarrayTypesBaseQuery = "type=listAttributes&mart=ENSEMBL_MART_ENSEMBL&virtualSchema=default&dataset=BIOMART_SPECIES_NAME_gene_ensembl&interface=default&attributePage=feature_page&attributeGroup=external&attributeCollection=microarray";
 
+
     /**
      * Downloads Ensembl-Microarray and Ensembl-Uniprot identifier mapping files for all species, if they exist.
      * @throws Exception
      */
-    protected void downloadData() throws Exception {
+    public void downloadData() throws Exception {
+
+        // Create directory where BioMart files will be stored.
+        Files.createDirectories(Paths.get(this.destination));
+        setProperties(EnsemblBiomartUtil.getProperties());
 
-        // TODO: Add unit tests
         // Get names of all organisms we add links and/or microarray data for.
-        // Species names are in biomart format (eg: hsapiens).
-        for (String biomartSpeciesName : getBiomartSpeciesNames()) {
-            logger.info("Retrieving Biomart files for " + biomartSpeciesName);
-            // Create directory where Biomart files will be stored.
-            new File(this.destination).mkdirs();
+        // Species names are in BioMart format (eg: hsapiens).
+        for (String speciesName : EnsemblBiomartUtil.getSpeciesNames()) {
+            System.out.println(speciesName);
```

Comment:
I think this `System.out.println` is a debugging statement, and it will be removed, right? ;)

Reply:
Yep I saw that last night after I pushed the commits. Good catch!